src/widget/sidebar_filler.c

```diff
@@ -0,0 +1,289 @@
+#include "sidebar_filler.h"
+
+#include "building/menu.h"
+#include "city/labor.h"
+#include "city/message.h"
+#include "city/population.h"
+#include "city/ratings.h"
+#include "city/view.h"
+#include "city/warning.h"
+#include "core/config.h"
+#include "core/direction.h"
+#include "core/lang.h"
+#include "core/string.h"
+#include "game/orientation.h"
+#include "game/settings.h"
+#include "game/state.h"
+#include "game/undo.h"
+#include "graphics/arrow_button.h"
+#include "graphics/graphics.h"
+#include "graphics/image.h"
+#include "graphics/image_button.h"
+#include "graphics/lang_text.h"
+#include "graphics/menu.h"
+#include "graphics/panel.h"
+#include "graphics/screen.h"
+#include "graphics/text.h"
+#include "graphics/window.h"
+#include "map/orientation.h"
+#include "scenario/property.h"
+#include "scenario/criteria.h"
+#include "sound/effect.h"
+#include "widget/city.h"
+#include "widget/minimap.h"
+#include "widget/sidebar.h"
```

Comment:
Sidebar_filler should ideally not have a dependency back on `sidebar`

Reply:
cause of the fun i had with constants; see prior comment